cookbooks/new_ping.md

```diff
@@ -0,0 +1,45 @@
+Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide to find out.
+
+Write Your Questions
+--------------------
+Do not try and implement new pings unless you know specifically what questions you're trying to answer. General questions about "How do users use our product?" won't cut it - these need to be specific, concrete asks that can be translated to data points. This will also make it easier down the line as you start data review.
+
+Create a Schema
+---------------
+Use JSON Schema to start with. See the examples schemas in the [Mozilla Pipeline Schemas repo](https://github.com/mozilla-services/mozilla-pipeline-schemas/).
+
+Start a Data Review
+-------------------
+Data review for new pings is a more complicated than when adding new probes. See [Data Review for Focus-Event Ping](https://bugzilla.mozilla.org/show_bug.cgi?id=1347266) as an example. Consider where the data falls in the [Data Collection Categories](https://wiki.mozilla.org/Firefox/Data_Collection).
+
+Submit Schema to mozilla-services/mozilla-pipeline-schemas
+-----------------------------------------
+Add parquet if you want. Make sure you add at least one example ping which the data can be validated against. Test your Parquet output in [Hindsight](https://pipeline-cep.prod.mozaws.net/) by testing it as an output plugin. See [Core ping output plugin](https://bugzilla.mozilla.org/attachment.cgi?id=8829626) for an example, where the parquet schema is specified as `parquet_schema`.
```

Comment:
I am not sure, that it should be in this manual, but it was not obvious for me which fields should I add to metadata (expecially before I got real results.).

As for Hindsight:
* It was not obvious which fields in parquet output should I change and which I shouldn't. (Also, turns out, that most of the fields are default, what if we will have them as a template in Hindsight?)
* After testing it will be really nice to get message like "You don't have any errors".
* As far as I remeber, "Deploy" button was available for me (but it is not supposed that I will deploy). I think, it is good to add to this manual the step to create bug to deploy.